### DISCUSSION TITLE
[BUG] Connect-PnPOnline - Use 'Update-Module -Name PnP.PowerShell'

### DISCUSSION THREAD

**Discussion by GTEM19889** [pnp/powershell#2912] [Q&A] [answered]
I have not used PnP.Powershell for a couple of years so when I returned to use it today using Connect-PnPOnline I received this message 
------ start -----
WARNING: 
A newer version of PnP PowerShell is available: 1.12.0.

Use 'Update-Module -Name PnP.PowerShell' to update.
Use 'Get-PnPChangeLog -Release 1.12.0' to list changes.

You can turn this check off by setting the 'PNPPOWERSHELL_UPDATECHECK' environment variable 
to 'Off'.
------- end ------
So I ran 'Update-Module -Name PnP.PowerShell' to update then closed PowerShell_Ise and tried again only to see the same message again. 

Tried several other things like re-installing then updating but it continues to suggest it is out of date.
Running 'Get-PnPChangeLog -Release 1.12.0' does not appear to do anything. (just returns back to prompt, presumably because I do not have that version)

### What is the version of the Cmdlet module you are running?
Get-Module -Name "PnP.PowerShell" -ListAvailable 
ModuleType Version
---------- -------
Manifest   1.9.0  
Manifest   1.8.0  
Manifest   1.7.0  
Manifest   1.6.0  

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows

**Comment by veronicageek** [comment, marked as answer]
@GTEM19889 - As you mention the PowerShell ISE, I believe you're still on PS5. Can you try run `Install-Module PnP.PowerShell -RequiredVersion 1.12.0 -Force`.

**Reply by GTEM19889** [reply]
Yes that appears to have done the trick.  Thank you for your help. (Not exactly sure why this is not a bug though)
